src/Codeception/Subscriber/Console.php

```diff
@@ -527,7 +527,9 @@ protected function detectWidth()
         ) {
             $this->width = (int) (`command -v tput >> /dev/null 2>&1 && tput cols`) - 2;
         } else if ($this->isWin() && (php_sapi_name() == "cli")) {
-            $this->width = 200;
+            exec('mode con', $output);
+            preg_match('/^ +.* +(\d+)$/', $output[4], $matches);
+            $this->width = (int) $matches[1] - 2;
```

Comment:
Maybe it was supposed to left some padding on the screen? 
I can't say why for sure in Linux version there was -2 but if it is there I found it is nicer so have some padding on the screen. Maybe it, maybe not. 

Just try by yourself how it looks better on Windows.